lib/module/xkcd.go

```diff
@@ -0,0 +1,70 @@
+package module
+
+import (
+	"fmt"
+	"github.com/davidscholberg/irkbot/lib/configure"
+	"github.com/davidscholberg/irkbot/lib/message"
+	"github.com/nishanths/go-xkcd"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const apiUrlFmtDefine = "https://relevantxkcd.appspot.com/process?%s"
+
+func Helpxkcd() []string {
+	s := "xkcd <search> - find a xkcd comic relevant to the search term"
+	return []string{s}
+}
+func get(apiURL string) (string, error) {
+	response, err := http.Get(apiURL)
+	if err != nil {
+		return "", err
+	}
+	defer response.Body.Close()
+	bodyBytes, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return "", err
+	}
+	bodyString := string(bodyBytes)
+	return bodyString, err
+}
+func fmtString(bodyString string) string {
+	bodyStrings := strings.Split(bodyString, "\n")
+	spacedStrings := strings.Fields(bodyStrings[2])
+	return spacedStrings[0]
+}
+func getXKCD(cfg *configure.Config, in *message.InboundMsg, actions *Actions) {
+	comic := "enter a search term, asshat"
```

Comment:
I did this since I did basically the same thing in 'interject.' Create the default message, and if there aren't any arguments, send that default message.